docs/content/en/docs/references/cli/_index.md

```diff
@@ -355,7 +355,7 @@ Options:
       --skip-tests=false: Whether to skip the tests after building
       --status-check=true: Wait for deployed resources to stabilize
   -t, --tag='': The optional custom tag to use for images which overrides the current Tagger configuration
-      --tail=true: Stream logs from deployed objects
+      --tail=false: Stream logs from deployed objects (true by default for `skaffold dev` and `skaffold debug`)
```

Comment:
This message is confusing though.  It shows that the default is false, except that here it is true since this is `skaffold debug`.